.github/workflows/build-ripunzip.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-20.04, macos-12, windows-2019]
+        os: [ubuntu-20.04, macos-14, windows-2019]
```

Comment:
This should be the oldest supported version, not the latest (we're building software we want to use on all supported OS, so that's easier to avoid compatibility issues with an old OS rather than a new one)

Reply:
Thanks for the heads up, and for the context, it makes a lot of sense